omnibus/files/server-ctl-cookbooks/infra-server/recipes/cleanup.rb

```diff
@@ -33,3 +33,17 @@
   recursive true
   action :delete
 end
+
+component_runit_service 'elasticsearch' do
+  action :disable
+end
+
+::Dir.glob(::File.join(node['private_chef']['user']['home'], '/elasticsearch/lib/log4j*')).each do |filename|
+  file filename do
+    action :delete
+  end
+end
+
+component_runit_service 'elasticsearch' do
+  action :enable
```

Comment:
Try without this resource declaration, since we already declare it above. If that does not work, keep the declaration but use `action :nothing`.

Reply:
made the changes. Thank you!